test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildRestoreTaskPackageSourceMappingTests.cs

```diff
@@ -0,0 +1,988 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.Frameworks;
+using NuGet.Packaging;
+using NuGet.ProjectModel;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace Msbuild.Integration.Test
+{
+    public class MsbuildRestoreTaskPackageSourceMappingTests : IClassFixture<MsbuildIntegrationTestFixture>
```

Comment:
Why is this file in the diff?  It doesn't appear to be a rename from another file, and adding PackageSourceMapping tests is out of scope for this PR (making tests run in parallel)

Reply:
Never mind, I didn't read the PR comment carefully enough, or the othe test file diff.